.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 package-lock.json
+*.tgz
```

Comment:
Is this for the build folder? If so - worth excluding whole build folder

Reply:
I did but though it was excluded still shows under the modified files on commit so I removed it.

Reply:
Make sure it's not already included in source control before your changes or you haven't added it in an earlier commit. But yeh that's fine ^^